platform/logging/appenders/Appenders.ts

```diff
@@ -0,0 +1,62 @@
+import { assertNever } from '../../lib/utils';
```

Comment:
Having `Appenders` and `Layouts` classes is "the best" factory-like thing I can think of at the moment to keep specific sub-class and sub-class config knowledge in a single place and keep code that uses it simple. It will probably evolve into something better later on.